src/data/complex/basic.lean

```diff
@@ -374,7 +374,7 @@ real.sqrt_nonneg _
 @[simp] lemma abs_eq_zero {z : ℂ} : abs z = 0 ↔ z = 0 :=
 (real.sqrt_eq_zero $ norm_sq_nonneg _).trans norm_sq_eq_zero
 
-@[simp] lemma abs_ne_zero {z : ℂ} : abs z ≠ 0 ↔ z ≠ 0 :=
+lemma abs_ne_zero {z : ℂ} : abs z ≠ 0 ↔ z ≠ 0 :=
```

Comment:
Why are you removing the `simp` attribute? To me this looks like a good simp-lemma.

Reply:
Linter complained:

```
/- The `simp_nf` linter reports: -/
/- SOME SIMP LEMMAS ARE NOT IN SIMP-NORMAL FORM.
see note [simp-normal form] for tips how to debug this.
https://leanprover-community.github.io/mathlib_docs/notes.html#simp-normal%20form
: -/
-- data/complex/basic.lean
#print complex.abs_ne_zero /- simp can prove this:
  by simp only [complex.abs_eq_zero, ne.def]
One of the lemmas above could be a duplicate.
If that's not the case try reordering lemmas or adding @[priority].
 -/
```

Reply:
@Ruben-VandeVelde, did you know that you can edit the labels on the PR from "changes requested" back to "request review"?  I am a novice contributor too, but I think this is perfectly good form, if you have addressed all the reviewers' comments.

Reply:
(To edit the labels, one must have write permissions to the repo, so I've just sent Ruben an invite.)

Reply:
Thanks!